src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,93 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union
+
+import curies
+import pandas as pd
+
+from utils import remove_angle_brackets, get_converter
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disorder',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_subtype_of_a_disorder',  # subtype of a disorder
+}
+ROBOT_TEMPLATE_HEADER = {
+    'mondo_id': 'ID',
+    'subset': 'A oboInOwl:inSubset',
+    'ordo_id': '>A oboInOwl:source',
+}
+
+
+def create_ordo_subsets_robot_template(
+    class_subsets_tsv_path: Union[str, Path], mondo_mappings_path: Union[str, Path], onto_config_path: Union[str, Path],
+    outpath: Union[str, Path],
+) -> pd.DataFrame:
+    """Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+    conv = curies.Converter = get_converter(onto_config_path)
+    
+    # Read: mondo.sssom.tsv
+    mondo_df = pd.read_csv(mondo_mappings_path, sep='\t', comment='#').rename(columns={
+        'subject_id': 'mondo_id',
+        'subject_label': 'mondo_label',
+    })
+    # - keep only exact matches
+    mondo_df = mondo_df[mondo_df['predicate_id'] == 'skos:exactMatch']
+
+    # Read: ORDO class subsets query results
+    df = pd.read_csv(class_subsets_tsv_path, sep='\t').rename(columns={
+        '?cls': 'ordo_id',
+        '?lbl': 'ordo_label',
+        '?grp': 'subset_ordo_class_id',
+        '?grp_lbl': 'subset_ordo_class_label',
+    }).sort_values(['ordo_id', 'ordo_label'])
+    # - remove duplicate rows where labels are a specific language variation, e.g. @en
+    #   this alternative works as of 2024/05/02, but is potentially trepidacious:
+    # df2 = df.drop_duplicates(subset=['ordo_id'], keep='last')
+    df['label_has_lang_variation'] = df['ordo_label'].map(lambda label: '@' in label)
+    df = df[~df['label_has_lang_variation']]
+    del df['label_has_lang_variation']  # this gets dropped later, but w/e
+
+    # Preprocess
+    for fld in ['ordo_id', 'subset_ordo_class_id']:
+        # - Strip annoying angle brackets from URIs
+        df[fld] = df[fld].map(remove_angle_brackets)
+        # - Compress class IDs
+        df[fld] = df[fld].map(lambda x: conv.compress(x))
```

Comment:
## Utilized `curies`
Since I needed to do 2/3 of the tasks in #515, I went ahead and did the last one, adding `curies`.

I also did a few other very tiny things that don't result in changes in outputs; comments and moving the location of some code. Easier to review by looking at [the last commit](https://github.com/monarch-initiative/mondo-ingest/pull/510/commits/ca64342c390e538b8d12cf425672cf8c5ff22977).

Reply:
Note: Nico approves.